etc/xcc-router/src/lib.rs

```diff
@@ -33,12 +39,14 @@ pub struct Router {
     nonce: LazyOption<u64>,
     /// The storage for the scheduled promises.
     scheduled_promises: LookupMap<u64, PromiseArgs>,
+    /// Account ID for the wNEAR contract.
+    wnear_account: AccountId,
 }
 
 #[near_bindgen]
```

Comment:
Wouldn't using `near_sdk` require us to use NEAR's preferred license...?

Reply:
It's a little unclear. Technically, the version of the sdk we are using has a GPL-3 license, but since then they have weakened it to be [MIT/Apache](https://github.com/near/near-sdk-rs/blob/master/near-sdk/Cargo.toml#L6), which I think is fine for our purposes. So we could change our dependency to a later commit (after their last release) to get to the point where they made the license change.

Reply:
Oh, great. If they have that clause, that makes me happy.